### PR TITLE
Web/CSS/CSS_flexible_box_layout/Basic_concepts_of_flexbox_basics_fix

### PR DESCRIPTION
**Pull request by PJW980921**
Dscription
ko, fr, pt-br, ru/docs/Web/CSS/CSS_flexible_box_layout/Basic_concepts_of_flexbox  에 basics 이미지들이 png로 되어있어서 그런지 엑박인 상태로 화면에 보여집니다.  

확장자 png를 svg로 수정했습니다.

스크린샷 2024-08-15 오후 7 07 16

### CODE REVIEW

**Review by cw118 — APPROVED** [review]
Looks good for `fr`, thank you @PJW980921 !

**Comment by PJW980921** [comment]
> Looks good for fr, thank you @PJW980921 !

You’re Welcome! Thank you for your comment!

**Review by 1ilsang — APPROVED** [review]
Looks good for `ko`. Thank you!

**Comment by PJW980921** [comment]
> Looks good for `ko`. Thank you!

감사합니다 😊

**Review by tristantheb — APPROVED** [review]

File: files/ru/web/css/css_flexible_box_layout/basic_concepts_of_flexbox/index.md [hunk 1]
```diff
@@ -24,21 +24,21 @@ slug: Web/CSS/CSS_flexible_box_layout/Basic_concepts_of_flexbox
 
 Если вы выберете `row` или `row-reverse`, ваша главная ось будет проходить в горизонтальном направлении **(inline direction)**.
 
-![If flex-direction is set to row the main axis runs along the row in the inline direction.](basics1.png)
+![If flex-direction is set to row the main axis runs along the row in the inline direction.](basics1.svg)
 
 Если вы выберете `column` или `column-reverse`, ваша главная ось будет проходить в вертикальном направлении **(block direction)**.
 
-![If flex-direction is set to column the main axis runs in the block direction.](basics2.png)
+![If flex-direction is set to column the main axis runs in the block direction.](basics2.svg)
```
Comment (leon-win) [inline files/ru/web/css/css_flexible_box_layout/basic_concepts_of_flexbox/index.md:31]:
```suggestion
![Если для flex-direction задано значение column, то главная ось проходит в вертикальном направлении.](basics2.svg)
```

File: files/ru/web/css/css_flexible_box_layout/basic_concepts_of_flexbox/index.md [hunk 2]
```diff
@@ -24,21 +24,21 @@ slug: Web/CSS/CSS_flexible_box_layout/Basic_concepts_of_flexbox
...
 
 ### Побочная ось
 
 Побочная ось проходит перпендикулярно главной оси, поэтому, если свойство `flex-direction` (главная ось) задано как `row` или `row-reverse`, побочная ось будет проходить в вертикальном направлении.
 
-![If flex-direction is set to row then the cross axis runs in the block direction.](basics3.png)
+![If flex-direction is set to row then the cross axis runs in the block direction.](basics3.svg)
```
Comment (leon-win) [inline files/ru/web/css/css_flexible_box_layout/basic_concepts_of_flexbox/index.md:37]:
```suggestion
![Если для flex-direction задано значение row, то побочная ось проходит в вертикальном направлении.](basics3.svg)
```

File: files/ru/web/css/css_flexible_box_layout/basic_concepts_of_flexbox/index.md [hunk 3]
```diff
@@ -24,21 +24,21 @@ slug: Web/CSS/CSS_flexible_box_layout/Basic_concepts_of_flexbox
...
 
 Если свойство `flex-direction` задано как `column` или `column-reverse`, побочная ось будет проходить в горизонтальном направлении.
 
-![If flex-direction is set to column then the cross axis runs in the inline direction.](basics4.png)
+![If flex-direction is set to column then the cross axis runs in the inline direction.](basics4.svg)
```
Comment (leon-win) [inline files/ru/web/css/css_flexible_box_layout/basic_concepts_of_flexbox/index.md:41]:
```suggestion
![Если для flex-direction задано значение column, то поперечная ось проходит в горизонтальном направлении.](basics4.svg)
```

File: files/ru/web/css/css_flexible_box_layout/basic_concepts_of_flexbox/index.md [hunk 4]
```diff
@@ -50,11 +50,11 @@ slug: Web/CSS/CSS_flexible_box_layout/Basic_concepts_of_flexbox
 
 Если свойству `flex-direction` задано значение `row` и вы работаете с английским языком, то начало главной оси будет слева, а конец главной оси – справа.
 
-![Working in English the start edge is on the left.](basics5.png)
+![Working in English the start edge is on the left.](basics5.svg)
```
Comment (leon-win) [inline files/ru/web/css/css_flexible_box_layout/basic_concepts_of_flexbox/index.md:53]:
```suggestion
![При работе на английском языке начало главной оси находится слева.](basics5.svg)
```

File: files/ru/web/css/css_flexible_box_layout/basic_concepts_of_flexbox/index.md [hunk 5]
```diff
@@ -50,11 +50,11 @@ slug: Web/CSS/CSS_flexible_box_layout/Basic_concepts_of_flexbox
...
 
 Если бы вы работаете с арабским языком, то начало главной оси будет справа, а конец главной оси – слева.
 
-![The start edge in a RTL language is on the right.](basics6.png)
+![The start edge in a RTL language is on the right.](basics6.svg)
```
Comment (leon-win) [inline files/ru/web/css/css_flexible_box_layout/basic_concepts_of_flexbox/index.md:57]:
```suggestion
![Для языков с направлением письма справа налево начало главной оси находится справа.](basics6.svg)
```

File: files/ru/web/css/css_flexible_box_layout/basic_concepts_of_flexbox/index.md [hunk 6]
```diff
@@ -119,7 +119,7 @@ slug: Web/CSS/CSS_flexible_box_layout/Basic_concepts_of_flexbox
 
 Если наш контейнер шириной 500 пикселей содержит три элемента шириной 100 пикселей каждый, для их размещения нам потребуется пространство шириной в 300 пикселей. Остаётся 200 пикселей свободного пространства. Если мы не изменим исходные значения этих свойств, флексбокс разместит это пространство за последним элементом.
 
-![This flex container has available space after laying out the items.](basics7.png)
+![This flex container has available space after laying out the items.](basics7.svg)
```
Comment (leon-win) [inline files/ru/web/css/css_flexible_box_layout/basic_concepts_of_flexbox/index.md:122]:
```suggestion
![В этом flex-контейнере остаётся место после размещения элементов.](basics7.svg)
```

**Review by leon-win — COMMENTED** [review]
@PJW980921 , thank you for fixing Russian version!

I have also added translations of image descriptions.

**Comment by PJW980921** [comment]
> @PJW980921 , thank you for fixing Russian version!
> 
> I have also added translations of image descriptions.

Even the translation into Russian is so good! 👍🏻

**Review by leon-win — APPROVED** [review]

**Comment by 1ilsang** [comment]
@clovislima Hi, Could you check this PR?

**Comment by PJW980921** [comment]
May I know when it will be merged? 
@1ilsang

**Comment by 1ilsang** [comment]
@mdn/yari-content-pt-br Hi, Can anyone help with this PR?

@PJW980921 This PR is without a Brazilian translation, so if they don't respond, I'll merge it in a week.

**Review by josielrocha — APPROVED** [review]
Approve for `@mdn/yari-content-pt-br`. 👍🏼

**Pull request merged**